report/collect_test.go

```diff
@@ -23,7 +23,7 @@ func TestSpecialContentTypes(t *testing.T) {
 		{
 			name:     "Javascript file",
 			fileName: "abc.js",
-			expected: "application/javascript",
+			expected: "text/javascript; charset=utf-8",
```

Comment:
We used an old stack and the new Go runtime uses this content type for JS files.